crates/uv-python/python/get_interpreter_info.py

```diff
@@ -401,15 +401,7 @@ def get_distutils_scheme():
             "data": scheme["data"],
         }
 
-    # By default, pip uses sysconfig on Python 3.10+.
-    # But Python distributors can override this decision by setting:
-    #     sysconfig._PIP_USE_SYSCONFIG = True / False
-    # Rationale in https://github.com/pypa/pip/issues/10647
-    use_sysconfig = bool(
-        getattr(sysconfig, "_PIP_USE_SYSCONFIG", sys.version_info >= (3, 10))
-    )
-
-    if use_sysconfig:
+    if use_sysconfig_scheme:
```

Comment:
I needed to move this out of `get_scheme` or `get_virtualenv` would fail on distutils import.